packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -189,30 +129,29 @@ export const LegalLinksWrapper = styled.span`
 
 export const LegalLink = styled(Link)`
   text-decoration: none;
-  font: ${cVar('typographyDesktopT200Strong')};
-  letter-spacing: ${cVar('typographyDesktopT200StrongLetterSpacing')};
-  text-transform: ${cVar('typographyDesktopT200StrongTextTransform')};
+  color: ${cVar('colorTextMuted')} !important;
 
   &:hover {
-    color: ${oldColors.gray[400]};
+    color: ${cVar('colorTextMuted')};
   }
 
   &:focus {
-    color: ${oldColors.gray[400]};
-  }
-
-  &:active {
-    color: ${oldColors.gray[500]};
+    color: ${cVar('colorTextMuted')};
```

Comment:
We should use some different color for hover/focus to indicate to the user that they can interact with the link, currently it stays the same color